custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js

```diff
@@ -1799,6 +1800,12 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, DTOp
                 'class': 'medium-col',
                 'value': renderMotherPhoneNumber
             },
+            // {
```

Comment:
is this good to remove?

Reply:
this is while removing featue flag..should i remove this?